docs/webapi/dragSlider.mustache

```diff
@@ -0,0 +1,9 @@
+Drag the srubber of a slider to a given position
```

Comment:
@PeterNgTr "srubber" should be "scrubber", or?

Reply:
fixed